deploy-tool/main.go

```diff
@@ -170,9 +190,15 @@ func generateAndDeployKeyPair(namespace, keyName string, keyMarshaler privateKey
 	secretArgument := fmt.Sprintf("--from-file=secret_key=%s", tempFile.Name())
 	kubectlCreate := exec.Command("kubectl", "-n", namespace, "create",
 		"secret", "generic", keyName, secretArgument, "--dry-run=client", "-o=json")
+
+	kubectlLabel := exec.Command("kubectl", "label", "-f-", "--dry-run",
+		"-o=json", "--local",
+		fmt.Sprintf("type=%s", keyType), fmt.Sprintf("ingestor=%s", ingestorName))
```

Comment:
Hmm that would work. I want to eventually stop making the secrets in terraform though since that's going to have to go away when the deploy-operator goes away. For example I'm not making the secret for the test ingestor in terraform.

I think if our goal is to move secret creation out of terraform we could potentially have this as an interim step?

Reply:
I believe moving secret creation out of Terraform means doing it in `deploy-operator`/`manifest-updater`, and that change would also obsolete `deploy-tool`. So adding this code doesn't actually help us make progress toward the automated secret generation/rotation world (because this new code will get deleted along with the rest of `deploy-tool`, and in the interim, I think it's more complicated than applying the label from Terraform, which I think would probably be a one line change compared to plumbing the value of `ingestorName` all the way from TF to here.